### ISSUE TITLE
Combo boxes or listboxes are not read in mozilla firefox

### ISSUE THREAD

**Issue by nvdaTrac** [nvaccess/test_nvdaTracImport#213]
**Reported by mike reiser on 8 Jul 2008 20:43 UTC**
In firefox, NVDA does not read the contents of comboboxes and other types of list box elements in forms.  Steps to reproduce: 

1.  Navigate with NVDA to a form that has a combo or listbox such as the combo boxes on the new ticket form on the nvda site.  

2.  When you land on the listbox, open it by pressing alt down arrow.  

3.  Notice that NVDA doesn't read the contents of the list boxes.  

Actual results: NVDA says nothing.  

Expected results:

NVDA should announce the items in the list box

**Comment by nvdaTrac** [comment]
**Comment by jteh on 9 Jul 2008 00:49 UTC**
I only had this problem for expanded combo boxes; collapsed combo boxes (i.e. not hitting alt+downArrow) worked fine. As far as I can tell, list boxes can't be expanded and I didn't have any problem with these either. Did you have problems with collapsed combo boxes and list boxes or was this just an assumption?

The problem with expanded combo boxes should be fixed in r2205. Please reopen if this does not fix it for you.
